config_examples/config.json

```diff
@@ -17,7 +17,7 @@
             "protocol": "https",
             "method": "POST",
             "headers": {
-                "ContenType": "application/xml",
+                "ContentType": "application/xml",
```

Comment:
Nice catch but I think it should be "Content-Type"

Reply:
Updated, Thanks! 🚀